src/test/java/com/exactpro/th2/codec/csv/TestCsvCodec.java

```diff
@@ -230,6 +235,40 @@ void skipsHeaderPublishing() {
                             () -> {
                                 assertEquals(1, message.getMetadata().getId().getSubsequenceCount());
                                 assertEquals(2, message.getMetadata().getId().getSubsequence(0));
+                                assertEquals("Csv_Message", message.getMetadata().getMessageType());
+                            },
+                            () -> assertEquals("1", getFieldValue(message, "A", () -> "No field A. " + message)),
+                            () -> assertEquals("2", getFieldValue(message, "B", () -> "No field B. " + message)),
+                            () -> assertEquals("3", getFieldValue(message, "C", () -> "No field C. " + message))
+                    )
+            );
+        }
+
+        @Test
+        void settingMessageTypeFromIncomingMessage() {
+            final var CUSTOM_TYPE = "csv_test_type";
```

Comment:
Why upper case for a local variable? I think we should follow the conventions and make it lower camel case